packages/editor/src/components/reusable-blocks-buttons/test/reusable-block-convert-button.js

```diff
@@ -1,56 +0,0 @@
-/**
```

Comment:
With the new implementation, it's hard to unit test these components with snapshots. I removed the tests because these are already covered in e2e tests.